src/main/java/gui/CreationModalView.java

```diff
@@ -0,0 +1,215 @@
+package gui;
+
+import javafx.event.EventHandler;
+import javafx.scene.control.Button;
+import javafx.scene.control.CheckBox;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Text;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class responsible for displaying a modal view for the creation of shots.
+ * @author alex
+ */
+public class CreationModalView extends ModalView {
+
+    private static final int width = 600;
+    private static final int height = 600;
+
+    private int numberOfCameras;
+
+    private VBox viewPane;
+    private List<CheckBox> cameraCheckboxes;
+    private TextField descripField;
+    private TextField nameField;
+    private NumberTextField startField;
+    private NumberTextField endField;
+    private Button creationButton;
+
+    private EventHandler<DirectorShotCreationEvent> shotCreationEventEventHandler;
+
+    /**
+     * Constructor.
+     * @param rootPane Pane to display modal on top of
+     * @param numberOfCamerasInTimeline Amount of cameras in timeline
+     * @param creationHandler Event handler for the creation of a shot
+     */
+    public CreationModalView(RootPane rootPane, int numberOfCamerasInTimeline,
+                             EventHandler<DirectorShotCreationEvent> creationHandler) {
+        super(rootPane, width, height);
+        this.numberOfCameras = numberOfCamerasInTimeline;
+        this.shotCreationEventEventHandler = creationHandler;
+        initializeCreationView();
+    }
+
+    /**
+     * Initialize and display the modal view.
+     */
+    private void initializeCreationView() {
+        this.viewPane = new VBox(20);
```

Comment:
I don't believe this is necessary. This can be found in the VBox documentation. (It's spacing between the children of the VBox pane) [https://docs.oracle.com/javase/8/javafx/api/javafx/scene/layout/VBox.html](url)